src/neofs_testlib/cli/neofs_adm/morph.py

```diff
@@ -127,7 +127,7 @@ def dump_hashes(self, rpc_endpoint: str) -> CommandResult:
         )
 
     def force_new_epoch(
-        self, rpc_endpoint: Optional[str] = None, alphabet: Optional[str] = None
+        self, rpc_endpoint: Optional[str] = None, alphabet_wallets: Optional[str] = None
```

Comment:
I'd split it into a separate commit, since it's a different (from set_config()) problem.

Reply:
done